### PR TITLE
Feature/update docker setup

### PR DESCRIPTION
**Pull request by LukBelter**
Updated Docker setup documentation.

### CODE REVIEW

File: www/user/howto/bco-demo.md [hunk 1]
```diff
@@ -2,21 +2,23 @@
...
-    --net=host \
-    --volume /etc/localtime:/etc/localtime:ro \
-    --volume /etc/timezone:/etc/timezone:ro \
-    --env BCO_OPTIONS='--db /tmp/bco/db --simulate' \
-    --detach \
-    --restart=always \
-    --log-driver=local \
-    -t \
-    openbaseorg/bco-demo:stable
+docker run \
```
Comment (LukBelter) [inline www/user/howto/bco-demo.md]:
```suggestion
sudo docker run \
```

File: www/user/howto/setup-docker.md [hunk 2]
```diff
@@ -26,19 +26,26 @@ sudo usermod -aG docker ${DEFAULT_USER}
...
 echo -e "allow_anonymous true\nlistener 1883" > $HOME/.mosquitto.conf && \
-sudo docker run \
-  --name mosquitto \
-  --publish 1883:1883 \
-  --volume $HOME/.mosquitto.conf:/mosquitto/config/mosquitto.conf \
-  --restart=always \
-  --log-driver=local \
-  -d \
-  eclipse-mosquitto
+docker run \
```
Comment (LukBelter) [inline www/user/howto/setup-docker.md]:
```suggestion
sudo docker run \
```

File: www/user/howto/setup-docker.md [hunk 3]
```diff
@@ -110,7 +117,8 @@ sudo usermod -a -G bco ${DEFAULT_USER}
 ```bash
 sudo docker run \
     --name bco \
-    --net=host \
+    --net=bco_net \
```
Comment (LukBelter) [inline www/user/howto/setup-docker.md]:
```suggestion
    --net=bco-net \
```

File: www/user/howto/setup-docker.md [hunk 4]
```diff
@@ -127,7 +135,7 @@ sudo docker run \
 ```bash
 sudo docker run \
     --name bco-device-manager-openhab \
-    --net=host \
+    --net=bco_net \
```
Comment (LukBelter) [inline www/user/howto/setup-docker.md]:
```suggestion
    --net=bco-net \
```

**Review by LukBelter — COMMENTED** [review]
Added missing sudo & changed bco_net into bco-net

File: www/user/howto/bco-demo.md [hunk 5]
```diff
@@ -2,21 +2,23 @@
...
+--net=bco-net \
+--publish 13781:13781 \
+--volume /etc/localtime:/etc/localtime:ro \
+--volume /etc/timezone:/etc/timezone:ro \
+--env BCO_OPTIONS='--db /tmp/bco/db --simulate --host mosquitto' \
+--detach \
+--restart=always \
+--log-driver=local \
+-t \
+openbaseorg/bco-demo:beta
```
Comment (DivineThreepwood) [inline www/user/howto/bco-demo.md]:
```suggestion
openbaseorg/bco-demo:stable
```

File: www/user/howto/bco-demo.md [hunk 6]
```diff
@@ -2,21 +2,23 @@
...
-    --log-driver=local \
-    -t \
-    openbaseorg/bco-demo:stable
+docker run \
+--name bco-demo \
+--net=bco-net \
+--publish 13781:13781 \
+--volume /etc/localtime:/etc/localtime:ro \
+--volume /etc/timezone:/etc/timezone:ro \
+--env BCO_OPTIONS='--db /tmp/bco/db --simulate --host mosquitto' \
```
Comment (DivineThreepwood) [inline www/user/howto/bco-demo.md]:
```suggestion
--env BCO_OPTIONS='--db /tmp/bco/db --simulate --host mqtt-broker' \
```

File: www/user/howto/setup-docker.md [hunk 7]
```diff
@@ -26,19 +26,26 @@ sudo usermod -aG docker ${DEFAULT_USER}
...
-sudo docker run \
-  --name mosquitto \
-  --publish 1883:1883 \
-  --volume $HOME/.mosquitto.conf:/mosquitto/config/mosquitto.conf \
-  --restart=always \
-  --log-driver=local \
-  -d \
-  eclipse-mosquitto
+docker run \
+--name mosquitto \
```
Comment (DivineThreepwood) [inline www/user/howto/setup-docker.md]:
```suggestion
--name mqtt-broker \
```

**Review by DivineThreepwood — CHANGES_REQUESTED** [review]
minor changes requested

File: www/user/howto/setup-docker.md [hunk 8]
```diff
@@ -26,19 +26,26 @@ sudo usermod -aG docker ${DEFAULT_USER}
 All following docker commands are performed as root via the `sudo` prefix since this is the default for the PI. In case you setup BCO in a rootless docker environment, then you have to remove the `sudo` prefix before performing each docker command. Otherwise it could happen that all containers are deployed to the docker v1 environment of the root user. 
 :::
 
+### Setup Network
+Setup a dedicated bco network for security reasons.
+```
+docker network create bco-net
```
Comment (DivineThreepwood) [inline www/user/howto/setup-docker.md]:
```suggestion
sudo docker network create bco-net
```

**Review by DivineThreepwood — APPROVED** [review]

**Pull request merged**